pandas/tseries/offsets.py

```diff
@@ -1863,35 +1852,29 @@ def apply(self, other):
         return result
 
     def get_year_end(self, dt):
-        if self.variation == "nearest":
-            return self._get_year_end_nearest(dt)
-        else:
-            return self._get_year_end_last(dt)
+        assert dt.tzinfo is None
 
-    def get_target_month_end(self, dt):
-        target_month = datetime(dt.year, self.startingMonth, 1,
-                                tzinfo=dt.tzinfo)
-        return shift_month(target_month, 0, 'end')
-        # TODO: is this DST-safe?
```

Comment:
non-issue b/c/ get_target_month_end is only ever called with `dt.tzinfo is None`